api/services/device_tags.go

```diff
@@ -30,28 +32,40 @@ func (s *service) CreateDeviceTag(ctx context.Context, uid models.UID, name stri
 		return NewErrTagLimit(DeviceMaxTags, nil)
 	}
 
-	if contains(device.Tags, name) {
-		return NewErrTagDuplicated(name, nil)
+	if contains(device.Tags, tag) {
+		return NewErrTagDuplicated(tag, nil)
 	}
 
-	return s.store.DeviceCreateTag(ctx, uid, name)
+	return s.store.DeviceCreateTag(ctx, uid, tag)
 }
 
-func (s *service) RemoveDeviceTag(ctx context.Context, uid models.UID, name string) error {
+// RemoveDeviceTag removes a tag from a device. UID is the device's UID and tag is the tag's name.
```

Comment:
This way to document the services is a usual way to do this or it is too much declarative?

Reply:
I like it.